content/english/_index.html

```diff
@@ -12,10 +12,80 @@
 {{< button external=true label="Visit the demo" link="https://demo.hedgedoc.org/" outline=true >}}
 {{< /banner >}}
 
+
 {{< features id="corefeatures" >}}
 
-{{< image-promo id="nocodimd" background="primary" title="Wasn't it called CodiMD?" >}}
+
+{{< icon-promo title="Let's get it started!" icon="rocket" background="secondary" right=true >}}
+Installing HedgeDoc is easy! We provide a precompiled build and a docker image. If you need help, then feel free to read our documentation.
```

Comment:
We don't "precompile" anything (it's bundling & minification). Maybe write "ready to use tar.gz"?

We also have another "call for documentation" in a section below. Maybe leave out the second sentence or write "For details, take a look at our install guides."

Reply:
done